journalwatch.py

```diff
@@ -480,11 +479,9 @@ def run():
     global config
     output = []
     config, patterns = parse_config_files()
-    try:
-        logging.basicConfig(level=getattr(logging, config.loglevel.upper()),
-                            format='%(asctime)s [%(levelname)s] %(message)s',)
-    except AttributeError:
-        raise JournalWatchError("Invalid loglevel {}".format(config.loglevel))
+    loglevel = 'DEBUG' if config.verbose else 'WARNING'
```

Comment:
You can just do `logging.DEBUG` and `logging.WARNING` here, and drop the `getattr` below.

Reply:
Ah, of course.